proxy/src/main/java/net/md_5/bungee/BungeeSecurityManager.java

```diff
@@ -14,7 +15,7 @@
 {
 
     private static final boolean ENFORCE = false;
-    private final Set<String> seen = new HashSet<>();
+    private final Set<String> seen = Collections.synchronizedSet( new HashSet<>() );
```

Comment:
ConcurrentHashMap.newKeySet()

Reply:
I really doubt it will make any difference.

Reply:
Sure but no reason to use the inferior version when a proper one exists.

Reply:
Alright